packages/data/src/api/dynamicNftMetadata/dynamicNftMetadata.mock.ts

```diff
@@ -0,0 +1,55 @@
+export const dynamicNftMetadataMockActive = {
+  nft_metadata: [
+    {
+      metadataId: 'dynamic.mintbase2.near:deadbeef2',
+      internalMetadataId: 0,
+      contractAddress: 'dynamic.mintbase2.near',
+      reference: 'https://arweave.net/reference2',
+      referenceBlob: {
+        media: 'https://arweave.net/media2',
+        history: [
+          'https://arweave.net/reference0',
+          'https://arweave.net/reference1',
+        ],
+      },
+      isActive: true,
+    },
+  ],
+  nft_metadata_aggregate: {
+    aggregate: {
+      count: 3,
+    },
+  },
+};
+
+export const dynamicNftMetadataMockInactive = {
+  nft_metadata: [
+    {
+      metadataId: 'dynamic.mintbase2.near:deadbeef0',
+      contractAddress: 'dynamic.mintbase2.near',
+      internalMetadataId: 0,
+      reference: 'https://arweave.net/reference0',
+      referenceBlob: {
+        media: 'https://arweave.net/media0',
+        history: [],
+      },
+      isActive: true,
```

Comment:
This mock value is called inactive, but has active = true... was that intended?
```suggestion
      isActive: false,
```